user-guides/asset-hosting/uploader.md

```diff
@@ -0,0 +1,57 @@
+---
+description: How to use Ocean Uploader
+---
+
+# Ocean Uploader
+
+### What is Ocean Uploader? 
+
+Uploader is designed to simplify the process of storring your assets on decentralized networks (such as [arweave](https://www.arweave.org/) and [filecoin](https://filecoin.io/)). It provides access to multiple secure, reliable, and cost-effective storage solutions in an easy to use UI and javascript library.
+
+### What decentralized storage options are available?
+
+Currently we support Arweave and we will soon be releasing Filecoin support. In the future we will also support multiple other options. 
+
+### How to store an asset on Arweave with the Uploader UI? 
+
+Ready to dive into the world of decentralized storage with Uploader? Let's get started:
+
+1. Wallet Check: First things first, let's make sure you've got some of that shiny polygon MATIC and WMATIC in your wallet. If you're coming up short, no worries! You can wrap your MATIC directly through the app.
```

Comment:
To make these steps easier, I would create an arcade to embed them all. It's super simple to use and to integrate. If you need help, please let me know

Reply:
yes please. If you can add it, that would it great 🙏🏾

Reply:
here you go: https://app.arcade.software/share/Hf5Fpy0dfJ4ame8OZOYl